src/api/lib/feature_switch/obs_repository.rb

```diff
@@ -3,27 +3,30 @@ module Repository
     # ObsRepository for active and inactive features based on YamlRepository having default values for each key in OBS
     #
     class ObsRepository < YamlRepository
+      # defaults need to be strings as we merge with yaml - which doesn't know symbols
       DEFAULTS = {
-        image_templates: true,
-        cloud_upload: false
+        'image_templates' => true,
```

Comment:
please use https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html and then we can keep it as symbol. We can use symbol in the YAML and the ruby YAML parse is able to parse it too without problems... said that I think HashWithIndifferentAccess better.